### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by Grinch**
- Adds /version (with /ver as an alias)
- Adds 'spout.command.version' permission to the 'version' command.
- Adds 'spout.command.batch' permission to the 'batch' command.
- Changes the server version output displayed on startup.

Signed-off-by: Steven Downer grinch@outlook.com

### CODE REVIEW

File: src/main/java/org/spout/engine/command/CommonCommands.java [hunk 1]
```diff
@@ -258,13 +260,20 @@ public void tp(CommandSource source, CommandArguments args) throws CommandExcept
 
 	@CommandDescription(aliases = "validate_movement", desc = "Toggle the validating of movement.")
 	@Filter(PlayerFilter.class)
-	public void validateInput(Player player, CommandArguments args) throws CommandException {
```
Comment (zml2008) [inline src/main/java/org/spout/engine/command/CommonCommands.java:261]: This should work because PlayerFilter.

Comment (Grinch) [inline src/main/java/org/spout/engine/command/CommonCommands.java:261]: Spout threw an error registering it at startup without having 'CommandSource'.

Comment (zml2008) [inline src/main/java/org/spout/engine/command/CommonCommands.java:261]: Fixed

**Comment by Grinch** [comment]
Updated to reflect @zml2008's recent fix in SpoutAPI for validateInput.

**Comment by macalinao** [comment]
Please also add /version [plugin]

**Comment by kitskub** [comment]
I agree with @simplyianm . Can you add that?

**Comment by Grinch** [comment]
@kitskub @simplyianm Perhaps in a future PR but as of right now I really have little intention to add such a feature. The goal was to stop relying on Vanilla's /version command and give Spout its own finally.

**Comment by kitskub** [comment]
Ok, sounds fine.

**Pull request merged**